emission/storage/timeseries/builtin_timeseries.py

```diff
@@ -378,9 +380,9 @@ def insert(self, entry):
             raise AttributeError("Saving entry %s for %s in timeseries for %s" % 
 		(entry, entry["user_id"], self.user_id))
         else:
-            logging.debug("entry was fine, no need to fix it")
+            logging.info("entry was fine, no need to fix it")
```

Comment:
Ditto

Reply:
Resolved in https://github.com/shankari/e-mission-server/pull/4/commits/3fbc72d73cd4d0e73f5ceb336e8c151e2da1ebc9